src/reference/features/testing/automate.md

```diff
@@ -1,39 +1,72 @@
 # Automate
 
-The `Automate` tab allows you to send requests in bulk. This feature provides a flexible way to bruteforce/fuzz certain parameters of requests using wordlists.
+The `Automate` tab allows you to send requests in bulk. This feature provides a flexible way to initiate bruteforce/fuzzing campaigns to rapidy modify certain parameters of requests using wordlists.
```

Comment:
`rapidly`, @Corb3nik maybe we could include https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker in the workspace to catch those? I dont like heavy spellcheck but it's nice to catch small errors.